lib/Interpreter/Interpreter.cpp

```diff
@@ -454,6 +469,15 @@ namespace cling {
 
   Transaction* Interpreter::Initialize(bool NoRuntime, bool SyntaxOnly,
                               llvm::SmallVectorImpl<llvm::StringRef>& Globals) {
+    // initialization is done by the host interpreter
```

Comment:
Possibly better:
```suggestion
    // Initialize the host interpreter to run host code, and pass device code to the device compiler using its `declare()` function
```
but not sure that's what you try to say here?

Reply:
Thanks, for me it's fine to return the `Transaction`, please go ahead.